### ISSUE TITLE
innounp@0.50: decompress error

### ISSUE THREAD

**Issue by Nthingxhere** [ScoopInstaller/Main#5826]
After wanting to install something via scoop this error is thrown how can I fix this? 
```powershell
Invoke-ExternalCommand : Das Argument für den Parameter "FilePath" kann nicht überprüft werden. Das Argument ist NULL
oder leer. Geben Sie ein Argument an, das nicht NULL oder leer ist, und führen Sie den Befehl erneut aus.
In C:\Users\[USER]\scoop\apps\scoop\current\lib\decompress.ps1:45 Zeichen:38
+     $Status = Invoke-ExternalCommand $7zPath $ArgList -LogPath $LogPa ...
+                                      ~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Invoke-ExternalCommand], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Invoke-ExternalCommand

Failed to extract files from C:\Users\[USER]\scoop\apps\innounp\0.50\innounp050.rar.
Log file:
  ~\scoop\apps\innounp\0.50\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=innounp%400.50%3a+decompress+error
```

**Comment by Nthingxhere** [comment]
Let me Know if I have to add something
Here are some infos done via winfetch
```
OS: Windows 11 Home [64-Bit]
Host: HP OMEN by HP Laptop 17-ck0xxx
Kernel: 10.0.22631.0
Motherboard: HP 88F7
Uptime: 2 hours 40 minutes                                  
Packages: 8 (scoop)
Shell: PowerShell v5.1.22621.2506
Resolution: 1707x960
Terminal: Windows Terminal
CPU: 11th Gen Intel(R) Core(TM) i7-11800H @ 2.304GHz
GPU: Intel(R) UHD Graphics
GPU: NVIDIA GeForce RTX 3070 Laptop GPU
Memory: 10,09 GiB / 31,65 GiB (31%)
Disk (C:): 869 GiB / 952 GiB (91%)
```

**Comment by HUMORCE** [comment]
Try update 7zip first.

**Comment by auxym** [comment]
I am getting the same error when attempting to install GraphicsMagick. I don't think it uses 7zip to unpack, rather I believe it calls `innounp.exe` to unpack the InnoSetup installer. The file innounp.log contains the following information:

```
Signature detected: Inno Setup Setup Data (6.3.0)
This is not directly supported, but i'll try to unpack it as version 5602
; Version detected: 6300
Critical error: The setup files are corrupted. Please obtain a new copy of the program.
```

**Comment by HUMORCE** [comment]
@auxym The issue you mentioned should be fixed by #6261.
